services/traction/api/endpoints/routes/v1/tenant/governance/schemas.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from starlette import status
+
+
+from api.endpoints.dependencies.db import get_db
+from api.endpoints.dependencies.tenant_security import get_from_context
+
+from api.services.v1 import ledger_service
+
+from api.db.repositories.tenant_schemas import TenantSchemasRepository
+from api.db.models.tenant_schema import TenantSchemaRead
+from api.endpoints.models.v1.governance import (
+    SchemasListResponse,
+    CreateSchemaPayload,
+    ImportSchemaPayload,
+)
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/", status_code=status.HTTP_200_OK, response_model=SchemasListResponse)
+async def get_tenant_schemas(
+    db: AsyncSession = Depends(get_db),
+) -> SchemasListResponse:
+    # copy of v0 implementation from endpoints/routes/tenant_admin
+    # this should take some query params, sorting and paging params...
+    wallet_id = get_from_context("TENANT_WALLET_ID")
+    schema_repo = TenantSchemasRepository(db_session=db)
+    tenant_schemas = await schema_repo.find_by_wallet_id(wallet_id)
+
+    response = SchemasListResponse(
+        items=tenant_schemas, count=len(tenant_schemas), total=len(tenant_schemas)
+    )
+    return response
+
+
+@router.post("/new", status_code=status.HTTP_200_OK)
```

Comment:
do not need "/new" here, just a post to the collection is fine and more consistent.  the "/import" is a special case.